docs/standard/frameworks.md

```diff
@@ -14,54 +14,31 @@ ms.assetid: 6ef56a2e-593d-497b-925a-1e25bb6df2e6
 
 # Target frameworks
 
-*Frameworks* define the objects, methods, and tools that you use to create apps and libraries. The .NET Framework is used to create apps and libraries primarily for execution on systems running a Windows operating system. .NET Core includes a framework that allows you to build apps and libraries that run on a variety of operating systems.
+Application Programming Interfaces (APIs) are contracts between software systems describing how their objects and methods can interact. Frameworks are made up of collections of APIs. When you target one or more frameworks in an app or library, you're specifying the sets and versions of APIs you'd like the app or library to use.
```

Comment:
I'm concerned that this topic *will be* accessed by many .NET beginners. I'm concerned that non-.NET devs may access this topic and not be clear on what we think these terms mean. If "target framework" represents API, then I'd define API first ... and I'd say what is meant by framework first. That paragraph was carefully constructed to ...

1. Define API
2. Say that frameworks are collections of APIs
3. Introduce the concept that "target[ing] frameworks" is specifying API and versions of API (and I think it should say 'one or more' to foreshadow xplat and cross-framework)

... so imo I don't think what you're proposing could be simpler than that. If we launch right into this with the sentence you propose, we fail to achieve (1) and (2) in the topic and just hope we don't lose the reader on the very first line.

> Might have to explain library, diff between spec and library, how an app or library that targets a spec ends up calling an actual library, etc.

I think this is covered implicitly .... *next* lol ... in the next paragraph, some of which I stole from u. lol 🚓 I think "library" is less overloaded and misunderstood, so I'm not too concerned about that term. I am concerned about adding "API specification," as that would introduce yet another term. "API specification" is implied by the description of .NET Standard's role in targeting frameworks, and "API specification" isn't used anywhere in the dotnet docs AFAICT.

Let me see what happens on the next commit. I'm just rolling thru here one para at a time. I'll keep this in mind when I look at the 2nd para next.